docker/build-docker-image.sh

```diff
@@ -1,9 +1,58 @@
 #!/bin/sh
-rm -f Dockerfile
-DIR_DOCKER_FILE=docker-centos-$1
-rm -rf $DIR_DOCKER_FILE
-mkdir  $DIR_DOCKER_FILE
-cp Dockerfile.template  $DIR_DOCKER_FILE/Dockerfile
-sed -i 's/{centosfrom}/'$1'/g' $DIR_DOCKER_FILE/Dockerfile
-sudo docker build $2 -t="erlang-rpm-build-"$1 $DIR_DOCKER_FILE 
-
+#
+#	
+#
+_docker_file="Dockerfile"
+_docker_temp="Dockerfile.template"
+_parameter1="$1"
+_parameter2="$2"
+_docker_dir="docker-centos-$_parameter1"
```

Comment:
Since you are there... can you please rename the parameters names :))) ?!

Something like:

```
centos_version="$1"
docker_params="$2"

docker_file="Dockerfile"
docker_template="Dockerfile.template"
docker_dir="docker-centos-$centos_version"
```
Thanks